container/deps/requirements.txt

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+accelerate==1.6.0
```

Comment:
@dmitry-tokarev-nv - for viz

Reply:
@dmitry-tokarev-nv Do I need to include this package in other places or here is enough?